### PR TITLE
Move Meshtasticator to Official Software and Add Documentation Pages

### PR DESCRIPTION
**Pull request by rcarteraz**
- Moved from community apps to software
- Add docs based on the repos docs
- Installed react player to use as video player (which is why there's package and lock file changes).

### CODE REVIEW

File: docs/software/meshtasticator/index.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+To get started with the interactive simulator, see the [interactive simulator usage guide](./interactive-sim).
+
+### Debugging and Visualization
+
+This simulator facilitates debugging multiple communicating nodes without needing physical devices.
+
+<div style={{ maxWidth: "800px", margin: "auto" }}>
+  <ReactPlayer
+    url="/img/software/meshtasticator/interactive-sim-video.mp4"
```
Comment (tekstrand) [inline docs/software/meshtasticator/index.mdx:49]: I am not familiar with our billing structure w/ Vercel but is there any risk of us hosting our own video files costing a lot of $ due to bandwidth?

File: docs/software/meshtasticator/interactive-sim.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+id: interactive-sim
+title: Interactive Simulator Usage Guide
+sidebar_label: Interactive Event Simulator
+sidebar_position: 2
+description: Interactive Simulator usage guide for simulating multiple Meshtastic instances with TCP-based communication and configurable pathloss models.
+---
+
+import ReactPlayer from "react-player";
+
+The Python script `_interactiveSim.py_` uses the [Linux native application of Meshtastic](https://meshtastic.org/docs/software/linux-native) to simulate multiple instances of the device software. These instances communicate using TCP via the script, simulating the LoRa chip. The simulator forwards messages from the sender to all nodes within range, based on their simulated positions and the selected pathloss model (see [Pathloss Model](#pathloss-model)). **Note:** Packet collisions are not yet simulated.
```
Comment (GUVWAF) [inline docs/software/meshtasticator/interactive-sim.mdx:11]: It seems these underscores cannot be used inside ` `.

**Review by GUVWAF — COMMENTED** [review]
Very nice, thank you for this! However, it seems that a general link to the repository is missing?

File: docs/software/meshtasticator/interactive-sim.mdx (hunk 2 above)
Comment (tekstrand) [reply docs/software/meshtasticator/interactive-sim.mdx:11]: Think you just have to escape them
`\_interactiveSim.py\_`

File: docs/software/meshtasticator/discrete-event-sim.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+id: discrete-event-sim
+title: Discrete Event Simulator Usage Guide
+sidebar_label: Discrete Event Simulator
+sidebar_position: 1
+description: Discrete Event Simulator usage guide, simulating radio communications with configurable parameters and node behaviors.
```
Comment (tekstrand) [inline docs/software/meshtasticator/discrete-event-sim.mdx:6]: Nitpick but the title doesn't need to be repeated in the description

File: docs/software/meshtasticator/interactive-sim.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+2. (Optional) Create a virtual environment.
+3. Install dependencies:
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+The simulator runs the Linux native application of Meshtastic firmware. You can use either [PlatformIO](https://meshtastic.org/docs/development/firmware/build) or [Docker](https://meshtastic.org/docs/software/linux-native#usage-with-docker) to run the firmware:
+
+- **Using PlatformIO:**
```
Comment (tekstrand) [inline docs/software/meshtasticator/interactive-sim.mdx:23]: I think it would be nice to have these as level 3 headings instead of bullet points. 🍺

**Review by tekstrand — APPROVED** [review]
No blockers, couple of nice to haves.

**Review by GUVWAF — APPROVED** [review]
Looks good to me, thanks again!

**Pull request merged**